taskt/Core/Automation/Commands/Folder Operation Commands/MoveCopyFolderCommand.cs

```diff
@@ -1,72 +1,72 @@
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Move Folder")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Copy Folder")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify whether you intend to move the folder or copy the folder. Moving will remove the folder from the original path while Copying will not.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select either **Move Folder** or **Copy Folder**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Folder Operation Type")]
+        [PropertyUISelectionOption("Move Folder")]
+        [PropertyUISelectionOption("Copy Folder")]
+        [InputSpecification("Specify whether you intend to move or copy the folder.")]
+        [SampleUsage("")]
+        [Remarks("Moving will remove the folder from the original path while Copying will not.")]
         public string v_OperationType { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the path to the source folder")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the folder.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\myfolder or [vTextFolderPath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Source Folder Path")]
+        [InputSpecification("Enter or Select the path to the original folder.")]
+        [SampleUsage(@"C:\temp\myfolder || {ProjectPath}\myfolder || {vOriginalFolderPath}")]
+        [Remarks("{ProjectPath} is the directory path of the current project.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)] 
         public string v_SourceFolderPath { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the directory to move/copy to")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the new path to the file.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\new path or [vTextFolderPath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Destination Folder Path")]
```

Comment:
In File Commands this description says "File Destination Path" so please change this one or that one for consistency.

Reply:
changed to "Destination File Path" in MoveCopyFileCommand.cs